src/applications/ds-v3-playground/utils/defineWebComponents.js

```diff
@@ -1,4 +1,4 @@
 import '@department-of-veterans-affairs/component-library/dist/main.css';
-import { defineCustomElementVaTextarea } from '@department-of-veterans-affairs/component-library/dist/components';
+import { defineCustomElements } from '@department-of-veterans-affairs/component-library';
```

Comment:
It looks like it was setup before to load individual components but now we're loading them all even if we aren't using them all?